From a9853fc10cc19859dc3d2d949c6d2d9907c25f67 Mon Sep 17 00:00:00 2001
From: Olivier Moysan <olivier.moysan@foss.st.com>
Date: Wed, 4 Jan 2023 10:10:30 +0100
Subject: [PATCH 318/529] ARM: dts: stm32: add vrefint calibration on stm32mp15

Describe vrefint calibration cell to be retrieved through bsec.

Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/283304
Signed-off-by: Olivier Moysan <olivier.moysan@foss.st.com>
Change-Id: I4edf2ceace42aea82394c0d5d927b816760c7f89
---
 arch/arm/boot/dts/stm32mp151.dtsi | 3 +++
 1 file changed, 3 insertions(+)

--- a/arch/arm/boot/dts/stm32mp151.dtsi
+++ b/arch/arm/boot/dts/stm32mp151.dtsi
@@ -1798,6 +1798,9 @@
 			part_number_otp: part-number-otp@4 {
 				reg = <0x4 0x1>;
 			};
+			vrefint: vrefin-cal@52 {
+				reg = <0x52 0x2>;
+			};
 			ts_cal1: calib@5c {
 				reg = <0x5c 0x2>;
 			};
